components/post_view/reaction/reaction.test.tsx

```diff
@@ -96,7 +96,7 @@ describe('components/post_view/Reaction', () => {
         const newActions = {...actions, getMissingProfilesByIds: jest.fn()};
         const props = {...baseProps, actions: newActions};
 
-        const wrapper = shallow(<Reaction {...props}/>);
+        const wrapper: any = shallow(<Reaction {...props}/>);
```

Comment:
Instead of defining the wrapper as any, you should be able to get the typing working here by defining the type on the `shallow` call instead

```suggestion
        const wrapper = shallow<Reaction>(<Reaction {...props}/>);
```
```